includes/admin/onboarding/steps/step-products.php

```diff
@@ -0,0 +1,202 @@
+								<?php
+							}
+
+							do_action( 'edd_price_field', null );
+							?>
+						</div>
+					</td>
+				</tr>
+
+				<tr class="edd-onboarding__product-variable-price">
+					<th scope="row"><label for="_edd_price_options_mode"><?php echo esc_html( 'Multi-option mode' ); ?></label></th>
+					<td>
+						<div class="edd-form-group">
+							<div class="edd-form-group__control">
+								<label class="edd-toggle edd-onboarding__multi-option-toggle" for="_edd_price_options_mode">
+									<?php echo EDD()->html->checkbox( array( 'name' => '_edd_price_options_mode', 'current' => null, 'class' => 'edd-form-group__input', ) ); ?>
+									<span><?php echo esc_html( apply_filters( 'edd_multi_option_purchase_text', __( 'Allows multiple price options to be added to your cart at once', 'easy-digital-downloads' ) ) ); ?></span>
+								</label>
+							</div>
+						</div>
+					</td>
+				</tr>
+				<tr class="edd-onboarding__product-variable-price no-table-row-padding">
+					<td colspan="2">
+						<div id="edd_variable_price_fields" class="edd_pricing_fields edd-onboarding__product-variable-price-fields">
+							<input type="hidden" id="edd_variable_prices" class="edd_variable_prices_name_field" value=""/>
+							<div id="edd_price_fields" class="edd_meta_table_wrap">
+								<div class="widefat edd_repeatable_table">
+									<div class="edd-price-option-fields edd-repeatables-wrap">
+										<div class="edd_variable_prices_wrapper edd_repeatable_row" data-key="1">
+											<?php do_action( 'edd_render_price_row', 1, array(), null, 1 ); ?>
+										</div>
+									</div>
+
+									<div class="edd-add-repeatable-row">
+										<button class="button button-secondary edd_add_repeatable"><?php echo esc_html( __( 'Add New Price', 'easy-digital-downloads' ) ); ?></button>
+									</div>
+								</div>
+							</div>
+						</div><!--end #edd_variable_price_fields-->
+					</td>
+				</tr>
+				<tr>
+					<th scope="row"><label><h3><?php echo esc_html( __( 'Add your first file', 'easy-digital-downloads' ) ); ?></h3></label></th>
+					<td><span alt="f223" class="edd-help-tip dashicons dashicons-editor-help" title="<?php echo esc_attr( __( 'Explanation for files creation.', 'easy-digital-downloads' ) ); ?>"></span></td>
+				</tr>
+				<tr class="edd-onboarding__product-files-row">
+					<td colspan="2">
+						<?php edd_render_files_field( null ); ?>
+					</td>
+				</tr>
+			</tbody>
+		</table>
+	</form>
+	<div class="edd-onboarding__product-created" style="display: none;">
+		<h1>🎉 <?php esc_html_e( 'Congratulations!', 'easy-digital-downloads' ); ?></h1>
+		<p><?php esc_html_e( 'You\'ve set up your store and your first product has been created.', 'easy-digital-downloads' ); ?></p>
+		<a href="" class="button button-primary button-hero edd-onboarding__edit-my-product"><?php esc_html_e( 'Edit My Product', 'easy-digital-downloads' ); ?></a>
```

Comment:
It might be helpful to wrap these two in their own container which could be styled with flexbox. I'm not sure it's necessary, but it might be nice.